scripts/test-uplink.sh

```diff
@@ -25,33 +25,37 @@ random_bytes_file () {
 random_bytes_file 2x1024      "$SRC_DIR/small-upload-testfile" # create 2kb file of random bytes (inline)
 random_bytes_file 5x1024x1024 "$SRC_DIR/big-upload-testfile"   # create 5mb file of random bytes (remote)
 
-uplink --config-dir "$GATEWAY_0_DIR"  mb "sj://$BUCKET/"
+UPLINK_DEBUG_ADDR="127.0.0.1:0"
```

Comment:
Let's disable debug-addr instead. Since this runs parallel with other tests, it may end up using up some other port that's needed.

Reply:
its choosing port when everything is started and its taking it only for one command but I have no problem with disabling it completely

Reply:
Yeah, but we potentially have two of these running in parallel. although, we haven't yet enabled IPv6 tests.

Reply:
done